src/checklist/checklist.py

```diff
@@ -39,13 +169,37 @@ def get_all_tests(self, requirements_only: bool = False):
     def get_test_areas(self):
         return self.test_areas
 
+    def to_yaml(self, output_path: str, no_preserve_format: bool = False, exist_ok: bool = False):
+        if not no_preserve_format:
+            raise NotImplementedError(
+                "Roundtripping is not yet implemented. If you want to dump the YAML file disregarding the original "
+                "formatting, use `no_preserve_format=True`."
+            )
+        elif not exist_ok and os.path.exists(output_path):
+            raise FileExistsError("Output file already exists. Use `exist_ok=True` to overwrite.")
+        YamlChecklistIO.write(output_path, self.content)
+
+    def to_csv(self, output_path, exist_ok: bool = False):
+        """Dump the checklist to three separate CSV files."""
+        if not exist_ok and os.path.exists(output_path):
+            raise FileExistsError("Output file already exists. Use `exist_ok=True` to overwrite.")
```

Comment:
Maybe it is more clear to write "Output directory already exists. Use `exist_ok=True` to overwrite."

Reply:
Done! Changed in latest commit 98e9373.